docs/releases/5.0.0/chapters/310-testing.html

```diff
@@ -497,9 +497,11 @@ <h4 id="finding-the-junit-platform-bundles">Finding the JUnit Platform bundles</
           url="file:///path/to/eclipse/";\
           location="${workspace}/cnf/cache/stable/EclipseLocal"
 </code></pre></div>    </div>
-    <p>Alternatively, it is not difficult to download the required (non-OSGi) modules from Maven Central and include them as-is on <code class="highlighter-rouge">-runpath</code>, or else (preferably) wrap them into bundles and include them in <code class="highlighter-rouge">-runrequires</code>/<code class="highlighter-rouge">-runbundles</code>.</p>
   </li>
 </ul>
+<p>Alternatively, it is not difficult to download the required (non-OSGi) modules from Maven Central and include them as-is on <code class="highlighter-rouge">-runpath</code>, or else (preferably) wrap them into bundles and include them in <code class="highlighter-rouge">-runrequires</code>/<code class="highlighter-rouge">-runbundles</code>. As of JUnit 5.6, the JUnit jars already have the OSGi metadata and so can be used as bundles direct from Maven Central.</p>
+
+<p>Also note that unfortunately, due to a bug in <code class="highlighter-rouge">biz.aQute.tester.junit-platform</code>, Bndtools 5.0 <em>does not work with JUnit 5.5+</em>. This has already been fixed in the main development branch and we expect the fix to be included in Bndtools 5.0.1 (to be released soon).</p>
```

Comment:
5.0.1 is possible. I have made a test build. But we still need to collectively discuss and agree on a 5.0.1 release. So is it safer to simply say the release after 5.0.